src/terminal/osc.zig

```diff
@@ -1928,6 +1936,21 @@ test "OSC: get/set clipboard with allocator" {
     try testing.expect(std.mem.eql(u8, "?", cmd.clipboard_contents.data));
 }
 
+test "OSC: clear clipboard" {
+    const testing = std.testing;
+
+    var p: Parser = .{ .alloc = testing.allocator };
+    defer p.deinit();
+
+    const input = "52;;";
+    for (input) |ch| p.next(ch);
+
+    const cmd = p.end(null).?;
+    try testing.expect(cmd == .clipboard_contents);
+    try testing.expect(cmd.clipboard_contents.kind == 'c');
+    try testing.expect(std.mem.eql(u8, "", cmd.clipboard_contents.data));
```

Comment:
`expectEqualStrings` would be better here IMHO.

Reply:
I literally just copied this from the previous one :bobr: